### DISCUSSION TITLE
Binding in which-key

### DISCUSSION THREAD

**Discussion by cipherodio** [nvim-neorg/neorg#381] [Q&A]
Hi first thanks for the awesome work. 😊

how can i bind this to which-key?

for example, i want this bind in which-key
```
{ leader .. "nn", "core.norg.dirman.new.note" },
```

i tried this but it's not working.
```
n = {
    name = "neorg",
    n = { "<cmd>core.norg.dirman.new.note", "new notes" },
```

**Comment by max397574** [comment]
just without the <cmd> thing
just the label

**Reply by cipherodio** [reply]
> just without the thing just the label

hi thanks, what do you  mean without the thing? 😅

**Reply by max397574** [reply]
```
n = {
    name = "neorg",
    n = {"new notes" },
```

**Reply by cipherodio** [reply]
> ```
> n = {
>     name = "neorg",
>     n = {"new notes" },
> ```

it only works because the default keybind for `core.norg.dirman.new.note` is `<leader> nn` 😅

if i changed it in which-key with this it doesn't work
```
o = {
    name = "neorg"
    n = { "new notes"},
}
```

**Reply by max397574** [reply]
yes
you can't set the mapping in which-key but only label it
